ydb/docs/en/core/dev/troubleshooting/performance/schemas/overloaded-shards.md

```diff
@@ -0,0 +1,29 @@
+# Overloaded shards
+
+## Description
```

Comment:
I thought we've removed such headers from other articles? Let's make it consistent.